dawn/src/dawn/SIR/SIR.h

```diff
@@ -302,89 +302,59 @@ struct Value : NonCopyable {
   static BuiltinTypeID typeToBuiltinTypeID(TypeKind type);
 
   /// Convert the value to string
-  std::string toString() const;
+  std::string toString() const { return std::string{typeToString(type_)}; }
```

Comment:
```suggestion
  std::string toString() const { return {typeToString(type_)}; }
```
this should also work, shouldn't it?

Reply:
I was being explicit, but yes that may trigger a double copy, better to use the implicit construct.